packages/core/helper-plugin/lib/src/components/GenericInput/tests/index.test.js

```diff
@@ -43,6 +44,29 @@ function setupNumber(props) {
   };
 }
 
+function setupDatetimePicker(props) {
+  const DATETIMEPICKER_FIXTURE_PROPS = {
+    type: 'datetime',
+    name: 'datetime-picker',
+    intlLabel: {
+      id: 'label.test',
+      defaultMessage: 'datetime picker',
+    },
+    value: null,
+    onChange: jest.fn(),
+    onClear: jest.fn(),
+    ...props,
+  };
+
+  const rendered = render(<ComponentFixture {...DATETIMEPICKER_FIXTURE_PROPS} />);
+
+  return {
+    ...rendered,
+  };
+}
+// Increase the jest timeout to accommodate long running tests
```

Comment:
```suggestion
/**
 * We extend the timeout of these tests because the DS 
 * DateTimePicker has a slow rendering issue at the moment. 
 * It passes locally, but fails in the CI.
 */
```

Okay, some feedback on your comment, what you've done is instead describe the code below, to me it's obvious we're increasing the Jest.Timeout to acommodate a long running test. What I don't know, is why are you doing this? Why is this important and why should I, developer X not just remove it? Can you see the difference in the information type i've presented in my comment? :)

Reply:
ok, thanks for your suggestion, I will replace my comment with your